app/modules/individuals/resources.py

```diff
@@ -364,3 +364,121 @@ def get(self, individual):
+        from flask_login import current_user
+
+        if not current_user or current_user.is_anonymous:
+            abort(code=401)  # anonymous cannot even request a merge
+
+        req = json.loads(request.data)
+        from_individual_ids = req  # assume passed just list, check for otherwise
+        if isinstance(req, dict):
+            from_individual_ids = req.get('fromIndividualIds', None)
+        if not isinstance(from_individual_ids, list):
+            abort(
+                success=False,
+                message='must pass a list of individuals IDs to merge from',
+                code=500,
+            )
+        if len(from_individual_ids) < 1:
+            abort(
+                success=False,
+                message='list of individuals IDs to merge from cannot be empty',
+                code=500,
+            )
+
+        # NOTE when merge conflict (DEX-514) is addressed, more potential args will be passed in
+
+        meets_minimum = False
+        # for which user does not have edit permissions
+        blocking_encounters = []
+        from_individuals = []
+        for from_id in from_individual_ids:
+            from_indiv = Individual.query.get(from_id)
+            if not from_indiv:
+                abort(
+                    success=False,
+                    message=f'passed from individual id={from_id} is invalid',
+                    code=500,
+                )
+            for enc in from_indiv.encounters:
+                if enc.current_user_has_edit_permission():
+                    meets_minimum = True
+                else:
+                    blocking_encounters.append(enc)
+            from_individuals.append(from_indiv)
+        for enc in individual.encounters:
+            if enc.current_user_has_edit_permission():
+                meets_minimum = True
+            else:
+                blocking_encounters.append(enc)
+
+        if not meets_minimum:
+            AuditLog.security_alert(
+                log,
+                f'requested unauthorized merge from {from_individuals}',
+                individual,
+            )
+            abort(code=403)
+
+        # all is in order for merge; is it immediate or just a request?
+        log.info(
+            f'merge wants {individual} from {from_individuals} (blocking: {blocking_encounters})'
+        )
+
+        if len(blocking_encounters) > 0:
+            block_ids = [str(enc.guid) for enc in blocking_encounters]
+            merge_request = None
+            try:
+                merge_request = individual.merge_request_from(*from_individuals)
+            except Exception as ex:
+                abort(
+                    blocking_encounters=block_ids,
+                    merge_request=True,
+                    message=str(ex),
+                    code=500,
+                )
+            if not merge_request:
+                abort(
+                    blocking_encounters=block_ids,
+                    merge_request=True,
```

Comment:
what is the merge_request field and why is it True if the merge reques failed?
I think this should be removed.

Reply:
`merge_request` is meant to distinguish between an **automatic merge** and one that was queued as a "merge request" (did not go thru automatically, but is pending vetting by stakeholders).  again, a complexity outlined in google docs.  basically it means "the merge did not happen, but _might_ happen in the future depending on whether anyone votes against it". 

also, fwiw, i suspect i will be modifying **what is returned** from all of the api calls a bit more once i finalize the next few PRs related to (1) real merge-request queuing; (2) conflict resolution.   so the exact return content should definitely be considered a work-in-progress that will change a bit more.   (e.g. it will for sure gain more details _about_ the queued merge-request, related to DEX-513 (working on currently).